core/lib/config/src/configs/da_dispatcher.rs

```diff
@@ -0,0 +1,58 @@
+use std::time::Duration;
+
+use serde::Deserialize;
+
+use crate::ObjectStoreConfig;
+
+#[derive(Clone, Debug, PartialEq, Deserialize)]
+pub struct DALayerInfo {
+    pub name: String,
+    #[serde(default)]
+    pub private_key: String,
```

Comment:
that makes total sense, I initially decided to go with that for the operator's simplicity because they could have just one config to follow, but it doesn't seem like a sustainable way in case there are more DA layers with radically different configs

so I moved the configs to the `hyperchain-da` and now they are imported together with the clients